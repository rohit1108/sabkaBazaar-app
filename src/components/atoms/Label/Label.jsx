import "./Label.scss";
const Label = ({id,label, hint, className}) =>{
    return (
        <label 
            htmlFor={id} 
            className={className}>
                <span>{label}</span>
                <span className="hint">{hint}</span>
        </label>
    )
}

export default Label;