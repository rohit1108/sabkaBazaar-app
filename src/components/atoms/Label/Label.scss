label{
    display: block;
    margin-bottom: -6px;
    .hint{
        color: black;
        font-size: 15px;
        font-weight: 700;
    }
}