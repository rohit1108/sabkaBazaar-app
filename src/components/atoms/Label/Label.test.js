//import './Input';
//import { getByRole, screen} from '@testing-library/react';
import Label from './Label';
import renderer from 'react-test-renderer';
import ReactDOM from 'react-dom';
//import { getAllJSDocTags } from 'typescript';

  it('renders the comp properly', () => {
    const divEl = document.createElement('div');
    ReactDOM.render(<Label />, divEl);
  });

