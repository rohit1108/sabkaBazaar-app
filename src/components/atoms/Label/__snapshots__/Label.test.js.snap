// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render input 1`] = `
<div
  className="label"
/>
`;

exports[`render label 1`] = `
<div
  className="label"
/>
`;

exports[`should have right Label comp snapshot 1`] = `
<div
  className="label"
/>
`;
